### PR TITLE
049 chamber botany

### PR DESCRIPTION
**Pull request by ChuChu42**
## About the Pull Request

049 now gets a mini botany in their chamber so they can put their skill im botany to a use

also added a shower head in 049's bathroom

image

## Why It's Good For The Game

049 spawns with botany skill of experienced by default, which then have no use by itself since 049 isnt able to do botany in his chamber

this PR adds a small botany room in his chamber, extending the literal brick of bushes to a full garden, giving 049 one more thing he can do to pass time

## Changelog

:cl:
add: 049 botany area
add: 049 shower head
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
good, will add content for the 049 player while he waits to do other stuff

**Comment by ChuChu42** [comment]
> good, will add content for the 049 player while he waits to do other stuff

in my other PR i also added a milk fridge so he can finally actually make bread

**Comment by Vizoplez** [comment]
Good content, but please prevent 049 from growing tomatoes/etc, else they'll TP out of their cell every round.

**Review by TheDarkElites — APPROVED** [review]
Looks fine to me.

**Comment by ChuChu42** [comment]
> Good content, but please prevent 049 from growing tomatoes/etc, else they'll TP out of their cell every round.

you cant easily mutate tomatos in this codebase

on a note: 049 can already easily make thermite

**Review by TheDarkElites — CHANGES_REQUESTED** [review]
Remove jumpsuits and backpacks from the lockers and other containers.

**Comment by ChuChu42** [comment]
> Remove jumpsuits and backpacks from the lockers and other containers.

i removed the jumpsuit and bags as well as added a rolling pin for cooking purposes

File: maps/site53/site53-1.dmm [hunk 1]
```diff
@@ -4057,7 +4075,7 @@
 /obj/effect/catwalk_plated,
 /obj/machinery/camera/network/lcz{
 	name = "SCP-500";
-	network = list("Heavy Containment Zone Network")
+	network = list("Heavy                Containment                Zone                Network")
```
Comment (TheDarkElites) [inline maps/site53/site53-1.dmm]: You shouldn't be randomly adding spaces, and camera networks are decided by the camera type not entered manually. For HCZ just use the HCZ preset for the cameras.

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

File: maps/site53/site53-1.dmm (hunk 1 above)
Comment (ChuChu42) [reply maps/site53/site53-1.dmm]: im not adding random spaces its some shit that strongDMM does

it looks fine ingame

Comment (ChuChu42) [reply maps/site53/site53-1.dmm]: also i didnt edit cameras in the first place

**Comment by ChuChu42** [comment]
strongdmm is on latest 8.1 build so im not sure why its making random spaces but it doesnt seem to be an issue ingame, i didnt add any cameras

**Comment by ChuChu42** [comment]
should be good

**Review by TheDarkElites — APPROVED** [review]
Gives 049 something to do and all issues were adressed

**Pull request merged**